pom.xml

```
@@ -244,6 +244,9 @@
         <plugin>
           <artifactId>maven-release-plugin</artifactId>
           <version>${maven-release-plugin.version}</version>
+          <configuration>
+            <releaseProfiles>!consume-incrementals</releaseProfiles>
```

If someone uses this patent they could have defined a `release` profile that would have been activated as the default and now isn't


```suggestion
            <releaseProfiles>release,!consume-incrementals</releaseProfiles>
```

---

Well unlike `plugin-pom` this POM is not setting

```xml
<useReleaseProfile>false</useReleaseProfile>
```

so https://github.com/apache/maven-release/blob/e8f77842fc3cf29d912fcdbb6525b409c971e44d/maven-release-manager/src/main/java/org/apache/maven/shared/release/phase/RunPerformGoalsPhase.java#L56-L66 should be active. Using 3.8.1 I checked in core

```bash
mvn help:active-profiles -P\!consume-incrementals -DperformRelease=true
```

which resulted in `consume-incrementals` being removed and `release-profile` being added as expected.